packages/react-components/src/templates/GroupProfileAbout.tsx

```diff
@@ -32,14 +35,33 @@ const GroupProfileAbout: React.FC<GroupProfileAboutProps> = ({
   leaders,
 
   membersSectionId,
-}) => (
-  <div css={styles}>
-    <GroupInformation tags={tags} description={description} />
-    <GroupTools calendarId={calendars[0] && calendars[0].id} tools={tools} />
-    <div id={membersSectionId}>
-      <GroupMembersSection teams={teams} leaders={leaders} />
+}) => {
+  const pmsEmails = leaders
+    .filter((pm) => pm.role.match(/project manager/i))
```

Comment:
Should not me fuzzy matching on enum/union

Reply:
Can you give me more details?

Reply:
Something like

```suggestion
const pmsEmails = leaders
    .filter(({role}) => role === "Project Manager")
    .map(({user: {email}}) => email);
```